src/zap/zapimport.cpp

```diff
@@ -2165,6 +2165,11 @@ DWORD ZapIndirectHelperThunk::SaveWorker(ZapWriter * pZapWriter)
     *(WORD *)(p + 2) = 0xC000;
     p += 4;
 
+    // orr r12, r12, #1
```

Comment:
This should be fixed by filling in the indirection with the high bit set at runtime instead